deps/oblib/src/lib/geo/ob_geo_utils.cpp

```diff
@@ -175,7 +175,7 @@ const char *ObGeoTypeUtil::get_geo_name_by_type(ObGeoType type)
       break;
     }
     case ObGeoType::GEOMETRYCOLLECTION:{
-      type_name = "GEOMETRYCOLLECTION";
+      type_name = "GEOMCOLLECTION";
```

Comment:
May not be modified more appropriate? 
In addition to error information, this public function will also affect the display of GIS WKT format, etc. And GEOMCOLLECTION stands for GEOMETRYCOLLECTION, will not affect the user's understanding of the error message.

Reply:
So not modify it.